feature-libs/asm/components/csagent-login-form/csagent-login-form.component.ts

```diff
@@ -1,20 +1,24 @@
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
-import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import {
+  UntypedFormBuilder,
+  UntypedFormGroup,
+  Validators,
+} from '@angular/forms';
 
 @Component({
   selector: 'cx-csagent-login-form',
   templateUrl: './csagent-login-form.component.html',
 })
 export class CSAgentLoginFormComponent implements OnInit {
-  csAgentLoginForm: FormGroup;
+  csAgentLoginForm: UntypedFormGroup;
```

Comment:
Overall, we're saying we are ok to wait until another major version in the future to add types for existing forms, if ever we decide to do it.

Reply:
ticket [CXSPA-1278](https://jira.tools.sap/browse/CXSPA-1278) was created.